bugbug/bug_features.py

```diff
@@ -182,6 +182,21 @@ class priority(object):
     def __call__(self, bug):
         return bug['priority']
 
+class comment_count(object):
+    def __call__(self, bug):
+        return field(bug, 'comment_count')
+
+
+class comment_length(object):
+    def __call__(self, bug):
+        comments = bug['comments']
```

Comment:
You could implement this more nicely with a [sum](https://docs.python.org/3/library/functions.html#sum).